ADAL/src/cache/ADTokenCacheItem+Internal.h

```diff
@@ -31,6 +31,7 @@
 
 @property (readwrite) NSMutableDictionary * additionalClient;
 @property (readonly) NSDictionary * additionalServer;
+@property (readonly) NSString *speInfo;
```

Comment:
nit: Can you add a comment on what this is? I'm sitting here a little puzzled by it.